pkg/terminal/command.go

```diff
@@ -1898,55 +1899,74 @@ func (c *Commands) sourceCommand(t *Term, ctx callContext, args string) error {
 	return c.executeFile(t, args)
 }
 
-var disasmUsageError = errors.New("wrong number of arguments: disassemble [-a <start> <end>] [-l <locspec>]")
+var disasmUsageError = errors.New("wrong number of arguments: disassemble [-a <start> <end>] [-l <locspec>] [-f <flavor>]")
 
 func disassCommand(t *Term, ctx callContext, args string) error {
-	var cmd, rest string
+	v := strings.FieldsFunc(args, func(c rune) bool {
```

Comment:
The argument to the `-l` is a location expression which could be a Go expression, you can't split the entire argument list, you have to do it one by one until you encounter `-l` and then stop.